megazords/full/android/dependency-licenses.xml

```diff
@@ -50,19 +50,19 @@ the details of which are reproduced below.
   </license>
   <license>
     <name>Apache License 2.0: askama</name>
-    <url>https://github.com/djc/askama/blob/main/LICENSE-APACHE</url>
+    <url>https://github.com/djc/askama/blob/master/LICENSE-APACHE</url>
```

Comment:
These changes look reversed? The 'main' branch exists in all these repos, but this patch is changing it to the old 'master'?

Reply:
Oh, hrm, TBH I didn't look very hard I just re-ran the script. Let me see if I can figure out what's happening here...